scripts/gui/thrive_style.css

```diff
@@ -1643,6 +1687,39 @@ div::-webkit-scrollbar-thumb:hover {
   font-size: 85%;
 }
 
+#ColourWheel {
+  position: absolute;
+  width: 200px;
+  height: 200px;
+  left: 15px;
+  top: 350px;
+  border-radius: 50%;
+  background-size: 200px 200px;
+  background-image: url("../../Textures/gui/colour_wheel.png");
+}
+
+#ColourDisplay {
+  position: absolute;
+  width: 10px;
+  height: 30px;
+  left: 0;
+  top: 435px;
+  border-radius: 10%;
+  background-size: 10px 30px;
+  background-color: rgb(255, 255, 255);
+}
+
+#ValueBar {
```

Comment:
Maybe this class should be more descriptive? We could have a million things in the GUI called a `value` bar.